.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish Release
+on:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      #- name: Set up JDK 11
```

Comment:
Why this commented out code? No need to setup the JDK after all? Either remove the commented out, or add inside the comment why its a good idea to keep the commented out stuff.

Reply:
Comment removed now